src/include/ndpi_main.h

```diff
@@ -150,6 +150,8 @@ extern "C" {
   void ndpi_set_risk(struct ndpi_detection_module_struct *ndpi_str,
 		     struct ndpi_flow_struct *flow, ndpi_risk_enum r,
 		     char *risk_message);
+  void ndpi_unset_risk(struct ndpi_detection_module_struct *ndpi_str,
```

Comment:
Not sure if we should export that function. It should be used with caution.

Reply:
Ok, let 's make it private. We can make it public again later, if necessary